### PR TITLE
rename and refactor contributing.rst to .md

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

**Comment by mmerickel** [comment]
is a file missing here?

**Comment by stevepiercy** [comment]
Not that I know of.

**Comment by mmerickel** [comment]
Err... the PR mentions a contributing.md which is not in this PR

**Comment by stevepiercy** [comment]
Oops. Got interrupted while doing this commit and overlooked it. Fixed in separate PR.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -2,42 +2,70 @@
...
 ----------------
 
 All features must be covered by functional tests and have example use.
 
+
 Preparing compatible browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Functional tests run on Firefox 43 and Selenium 2.56. Below are instructions for OSX:
+Functional tests run on Firefox 43 and Selenium 2.56. Below are instructions
```
Comment (tisdall) [inline CONTRIBUTING.rst]: This says "Firefox 43" but below it says "45"

Comment (stevepiercy) [inline CONTRIBUTING.rst]: Would you open a new issue for this? @miohtama would know the minimum requirements. I merely renamed the file and added some spit and polish. Thank you!

**Pull request merged**